packages/slate-editor/src/components/SearchInput/SearchInput.tsx

```diff
@@ -92,6 +97,13 @@ export function SearchInput<T = unknown>({
 
     useRootClose(rootRef, () => setOpen(false));
 
+    useEffect(() => {
+        const unlisten = EventsEditor.addEventListener(editor, 'contact-dialog-searched', () =>
```

Comment:
Yeah, I've seen it but it expects `EditorRef` and not the `Editor` object directly 🤔 So I think it's meant to be used for the consumer code and not within the editor.

Reply:
Which reminds me that I should probably use that in our codebase instead of accessing `events` directly.

Reply:
> Which reminds me that I should probably use that in our codebase instead of accessing events directly.

Hm... 

No, I don't think it'll work. `useEditorEvents()` called from the app will establish listener for events generated somewhere else. And in this case, we want the opposite: to listen inside the editor for events  generated somewhere else. Right?

Reply:
Yes, indeed 👍